src/Altinn.App.Core/Features/IFormDataValidator.cs

```diff
@@ -37,6 +37,7 @@ public interface IFormDataValidator
     /// <param name="instance"></param>
     /// <param name="dataElement"></param>
     /// <param name="data"></param>
+    /// <param name="language">Current language used by the user</param>
```

Comment:
```suggestion
    /// <param name="language">The currently used language by the user (or null if not available)</param>
```

Sync it with the rest of the language descriptions